tests/describe/t6120.c

```diff
@@ -104,7 +104,7 @@ static void commit_and_tag(
 
 	cl_git_append2file("describe/file", "\n");
 	
-	cl_git_pass(git_index_add_bypath(index, "describe/file"));
+	cl_git_pass(git_index_add_bypath(index, "file"));
```

Comment:
I'd love to hear some details why this is necessary. Tests passed before, so why is it needed now?

Reply:
The path given to `git_index_add_bypath` is relative to the root of the repository. That `describe/file` path is relative to the root of the sandbox directory, hence if I add the missing `cl_git_pass` I rightfully get an error that `$SANDBOX/describe/describe/file doesn't exist`. The path is thus changed to be made relative to the repository, which makes the failure go away and "restore" the test.

Hence the test doesn't work as expected, because it does not care about `file` being added or not to the index, which makes dubious in my eyes.

Reply:
I'll put it in the commit message then 😉. I still feel like there's something dubious going on though, but this is about coverity.